docs/crd_mysql_restore_job.md

```diff
@@ -0,0 +1,47 @@
+RestoreJob
+==========
+
+`RestoreJob` is a custom resource definition (CRD) that representsa Point-in-Time Recovery (PiTR)
+job that targets a given [`Cluster`](crd_mysql_cluster.md).
+
+Restoration fails if the appropriate dump or binlog file is lost, and the operator does nothing.
+
+| Field        | Type                                  | Description                                |
+| ------------ | ------------------------------------- | ------------------------------------------ |
+| `apiVersion` | string                                | APIVersion.                                |
+| `kind`       | string                                | Kind.                                      |
+| `metadata`   | [ObjectMeta]                          | Standard object's metadata.                |
+| `spec`       | [RestoreJobSpec](#RestoreJobSpec)     | Configuration for PiTR.                    |
+| `status`     | [RestoreJobStatus](#RestoreJobStatus) | Most recently observed status of the PiTR. |
+
+RestoreJobSpec
+--------------
+
+| Field               | Type   | Description                                                  |
+| ------------------- | ------ | ------------------------------------------------------------ |
+| `targetClusterName` | string | Target [`Cluster`](crd_mysql_cluster.md) name.               |
```

Comment:
I think targetCluster should be dynamically created.
Restoring to the existing cluster is dangerous.  It may risk precious data.